src/Interfaces/WiFiInterface.cpp

```diff
@@ -23,7 +23,7 @@ COPYRIGHT (c) 2017-2019 Mike Dunston
 #include <IPAddress.h>
 #include "WebServer.h"
 #include <esp_log.h>
-#include <esp_wifi_internal.h>
+//#include <esp_wifi_internal.h>
```

Comment:
This file is still valid and used for idf v3.3, but it can simply be removed rather than commented.

Reply:
i'm dropping this include in the next push to development branch, marking this resolved.